### PR TITLE
Platform efm32gg

### PR DESCRIPTION
**Pull request by perigoso**
Add support for the efm32 giant gecko family

### CODE REVIEW

File: build_targets/families.py [hunk 1]
```diff
@@ -137,10 +137,12 @@ def source(self) -> List[str]:
             'emu.c',
             'cmu.c',
             'systick.c',
+            'gpio.c',
         )
 
     def c_flags(self) -> List[str]:
         return [
+            '-g',
```
Comment (FFY00) [inline build_targets/families.py]: You probably didn't mean to commit this, right?

File: build_targets/families.py [hunk 2]
```diff
@@ -175,10 +175,7 @@ def ld_flags(self) -> List[str]:
         ]
 
     def dependencies(self) -> List[BuildDependency]:
-        cmsis_deps = [
+        return [
             CMSISDependency(components=['Core']),
             CMSISDeviceEFM32GG12BDependency(),
         ]
```
Comment (FFY00) [inline build_targets/families.py]: You removed the tinyusb dep here?

File: build_targets/families.py (hunk 1 above)
Comment (perigoso) [reply build_targets/families.py]: yup, leftovers, I should add this to a if block with the debug option

File: build_targets/families.py (hunk 2 above)
Comment (perigoso) [reply build_targets/families.py]: leftovers from debugging the build system, ups

**Comment by perigoso** [comment]
I'm confused, I think you reviewed old files, couldn't find any of those leftovers, was it in a specific commit?

**Comment by FFY00** [comment]
Yes, I did the reviews per commit but Github UI is awful showing that :/

**Comment by perigoso** [comment]
Should be clean now

**Comment by perigoso** [comment]
Should be all good now

**Review by FFY00 — APPROVED** [review]

**Pull request merged**